Classes/corrector.rb

```diff
@@ -0,0 +1,11 @@
+class Corrector
+  attr_accessor :name
+
+  def initialize(name)
+    @name = name
+  end
+
+  def correct_name
+    @name.length > 10 ? @name[0..9].capitalize : @name.capitalize
```

Comment:
- [ ] It will be faster and cleaner to have something like this:
```ruby
name.capitalize[0,10]
```
This way if the word length is less than 10, it will show the capitalized word anyway, kindly change it 😄

Reply:
Wow, faster. Thanks Man